src/ds/ds-private.h

```diff
@@ -60,6 +60,10 @@ namespace librealsense
         const uint8_t DS5_THERMAL_COMPENSATION              = 0xF;
         const uint8_t DS5_EMITTER_FREQUENCY                 = 0x10;
         const uint8_t DS5_DEPTH_AUTO_EXPOSURE_MODE          = 0x11;
+
+        const uint8_t DS5_HKR_PVT_TEMPERATURE               = 0x15;
```

Comment:
Shouldn't these be in d500-private??

Reply:
Sounds like it, @remibettan ?

Reply:
we agreed this is ok to for that to remain here, so that we do not override the value here for some other common command

Reply:
I don't like it. The code dependencies should reflect actual products. D400 shouldn't know about D500 etc.
Common commands need to take into account all products -- "ensuring it" by putting all product codes here is not right.